RELEASE-NOTES.rst

```diff
@@ -18,6 +18,220 @@
 Release notes
 =============
 
+v0.10.0
+-------
+
+User interface
+--------------
+
+Public interface
+~~~~~~~~~~~~~~~~
+
+-  Keeps the active tab on the document or person detailed view on page
+   reloads.
+
+Professional interface
+~~~~~~~~~~~~~~~~~~~~~~
+
+-  Renders the language menu with the same look as in the public
+   interface:
+
+   -  Uses the same icons.
+   -  Removes the current language.
+   -  Avoids to translate the language menu.
+   -  Adds a divider to separate the language menu from the help link.
+
+-  Takes the entire screen width. This is useful for the improvement of
+   the editor.
+-  Improves the document editor:
+
+   -  Sets a max width for selects and inputs.
+   -  Sets bold font weight for titles.
+   -  Hides fields with unique value (as ``bf:Place``).
+   -  Displays fields inline and makes this configurable through a HTML class
+      in the JSON schema.
+   -  Adds HTML class in the JSON schema to fix max width and title font
+      size.
+
+-  Implements interface components to import bibliographic records from
+   external sources through the web (SRU protocol). The librarian searches in a
+   regular search interface for the desired record, using a simple query
+   (keywords for the author, title, date, IDs…), selects a record, gets a
+   preview in two formats (the RERO ILS JSON rendered in HTML, and MARC). Once
+   the desired record is identified, through an *Import* button, the record is
+   loaded into the document editor to be modified and then added to the
+   catalog. At this stage, only the BnF SRU service is implemented.
+-  Fixes the redirection to the parent document after the deletion of an item.
+   This behaviour has been generalized to every parent/child resource
+   relationship through a modification in the routing configuration.
+-  Allows to receive an issue (serials) through the professional
+   interface (see the acquisition section, below). The workflow begins on the
+   holding detailed view and then continues on a dedicated view for serial
+   issues. The *Quick receipt* avoids the burden to manually create an item,
```

Comment:
I read online that it is better to use `Quick Receive`. `Quick receipt` tend to be  receiving a signal on an electronic device

Reply:
`avoids the burden to manually create an item when possible.` rephrase if possible.
receives automatically a regular issue and creates its corresponding item?

Reply:
@BadrAly For the receive/receipt you have to check with @pronguen.

Reply:
I think it is not that important in the release note... we will have to decide for the interface